modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -4677,10 +4677,9 @@ bool GDScriptAnalyzer::get_function_signature(GDScriptParser::Node *p_source, bo
 
 		for (const MethodInfo &E : methods) {
 			if (E.name == p_function) {
-				function_signature_from_info(E, r_return_type, r_par_types, r_default_arg_count, r_static, r_vararg);
-				r_static = Variant::is_builtin_method_static(p_base_type.builtin_type, function_name);
```

Comment:
I wanted to bring attention to this. I removed this line entirely, because it seems to me that the static information should already be coming from `function_signature_from_info()`, and the tests pass without this.

So either the test coverage isn't good enough, or this was redundant!

Reply:
It looks like it's the same thing.

https://github.com/godotengine/godot/blob/49243a9a9816b334002f7d73d9e940d00b49158d/core/variant/variant_call.cpp#L1344-L1349

https://github.com/godotengine/godot/blob/49243a9a9816b334002f7d73d9e940d00b49158d/core/variant/variant_call.cpp#L1405-L1407

Reply:
I figured so too, but because this part of core is pretty foreign to me, I don't know whether it checks for a few more things that `function_signature_from_info()` might not. So at least it's documented here as a possibility.